examples/Spirit/spirit_utils.cc

```diff
@@ -593,6 +641,89 @@ double calcTorqueInt(
   return act_int;
 }
 
+template <typename T>
+void AddWorkCost(drake::multibody::MultibodyPlant<T> & plant,
+                 dairlib::systems::trajectory_optimization::Dircon<T>& trajopt,
+                 double cost_work_gain,
+                 double work_constraint_scale,
+                 double efficiency){
+  auto velocities_map = multibody::makeNameToVelocitiesMap(plant);
+  auto actuator_map = multibody::makeNameToActuatorsMap(plant);
+  int n_q = plant.num_positions();
+
+
+  // Vector of new decision variables
+  std::vector<drake::symbolic::Variable> power_pluses;
+  std::vector<drake::symbolic::Variable> power_minuses;
+  double Q = 0;
```

Comment:
stylistic choice, but since we never want Q =0 maybe just declare it without value?